inc/functions/tax-project.php

```diff
@@ -38,7 +38,8 @@ function project_hide_slug() {
  * @return string The redirect_to value
  */
 function bulk_translate_projects_by_request_posts( $redirect_to, $action, $post_ids ) {
-    $languages = sanitize_text_field( wp_unslash( $_GET['translationmanager_bulk_languages'] ?? '' ) );
+    /** @var array<array-key, mixed> $languages */
+    $languages = \filter_input( INPUT_GET, 'translationmanager_bulk_languages', FILTER_SANITIZE_STRING, FILTER_REQUIRE_ARRAY );
```

Comment:
Hi @devbanana, sorry for chiming in, but `FILTER_SANITIZE_STRING` is deprecated on PHP 8.1. That's why @Narek13 changed this in #39 as part of [TM-259].

Reply:
@tyrann0us I see, I didn’t realize that was deprecated. Then since it’s expecting an array of IDs, why not `FILTER_SANITIZE_NUMBER_INT`? Would that be acceptable here?

Reply:
Haven't tested this, but `FILTER_SANITIZE_NUMBER_INT` should indeed work.
```suggestion
    $languages = \filter_input( INPUT_GET, 'translationmanager_bulk_languages', FILTER_SANITIZE_NUMBER_INT, FILTER_REQUIRE_ARRAY );
```
@shvlv, please scream if this is nonsense. 😉

Reply:
Yes, it should work. I will try to test later.

Reply:
I confirm, it works.